indico/htdocs/js/indico/jquery/principalfield.js

```diff
@@ -50,15 +50,19 @@
         _add: function _add(people) {
             var self = this;
 
+            function comparePersons(first, second) {
+                /* Compare using EventPerson.id or the e-mail if an id is not available. */
```

Comment:
Not a fan of comments that simply state what the code does if there's no hidden caveats.